models/reviews.model.js

```diff
@@ -43,12 +43,38 @@ exports.selectCommentsByID = (review_id) => {
     const commentsbyID = db.query(`SELECT * FROM comments WHERE review_id = $1;`, [review_id])
 
     return Promise.all([reviewsByID, commentsbyID]).then(([reviewsResults, commentsResults]) => {
-        console.log(commentsResults.rows)
-        
         if (reviewsResults.rows.length >0 && commentsResults.rows.length >= 0) {
             return commentsResults.rows
         } else {
             return Promise.reject({status: 404, msg: "Review does not exist"})
         }
     })
+}
+
+exports.insertCommentByID = (review_id, username, body) => {
+    if (typeof body != "string" || body.length <1) {
+        return Promise.reject({status: 400, msg: "Invalid comment"})
+    }
+    return db.query(`SELECT review_id FROM reviews WHERE review_id=$1`, [review_id])
+    .then((result) => {
+        if (result.rows.length === 0) {
+            return Promise.reject({status: 404, msg: "Review not found"})
+        }
+        else {
+            return db.query(`SELECT username FROM users WHERE username = $1`, [username])
```

Comment:
to avoid a bunch of nesting - you can do this return on line 64 but then chain a then block onto the then block that was initialised on line 59

Remember all we're doing is returning the promise and passing the result onto the next then block, this should make it look nicer